sopel/coretasks.py

```diff
@@ -122,6 +131,21 @@ def _execute_perform(bot):
         bot.write((command,))
 
 
+@plugin.event(events.ERR_NICKNAMEINUSE)
+@plugin.thread(False)
+@plugin.priority('high')
+@plugin.unblockable
+def on_nickname_in_use(bot, trigger):
+    LOGGER.error(
+        'Nickname already in use! '
+        '(Nick: %s ; Sender: %s ; Args: %r)',
```

Comment:
If there's no technical reason to have these spaces, I think they look weird in log output.

```suggestion
        '(Nick: %s; Sender: %s; Args: %r)',
```